com.unity.ml-agents/Runtime/Actuators/IActionReceiver.cs

```diff
@@ -57,12 +66,40 @@ public override int GetHashCode()
                 return (ContinuousActions.GetHashCode() * 397) ^ DiscreteActions.GetHashCode();
             }
         }
+
+        /// <summary>
+        /// Packs the continuous and discrete actions into one float array.  The array passed into this method
+        /// must have a Length that is greater than or equal to the sum of the Lengths of
```

Comment:
Should we check this early? Or just rely on Array.Copy() to do the bounds check?

Reply:
I can add a debug assert.  How does that sound?

Reply:
Sounds good.